server/src/testFixtures/java/io/crate/testing/TestingHelpers.java

```diff
@@ -132,16 +132,25 @@ private static boolean printObject(PrintStream out, boolean first, Object o) {
                 }
             }
             out.print("]");
+        } else if (o instanceof List<?> list) {
+            out.print("[");
+            for (int i = 0; i < list.size(); i++) {
+                printObject(out, true, list.get(i));
+                if (i < list.size() - 1) {
+                    out.print(", ");
+                }
+            }
+            out.print("]");
         } else if (o.getClass().isArray()) {
             out.print("[");
             boolean arrayFirst = true;
             for (int i = 0, length = Array.getLength(o); i < length; i++) {
                 if (!arrayFirst) {
-                    out.print(",v");
```

Comment:
Pretty sure this is just a bug, as it didn't get triggered anywhere until this change.